infrastructure/main.tf

```diff
@@ -3,24 +3,10 @@ provider "azurerm" {
 }
 
 locals {
-  app_full_name = "${var.product}-${var.component}"
-  aseName = "core-compute-${var.env}"
 
-  local_env = "${(var.env == "preview" || var.env == "spreview") ? (var.env == "preview" ) ? "aat" : "saat" : var.env}"
-  local_ase = "${(var.env == "preview" || var.env == "spreview") ? (var.env == "preview" ) ? "core-compute-aat" : "core-compute-saat" : local.aseName}"
+  vaultName = "${var.core_product}-${var.env}"
 
-  previewVaultName = "${var.core_product}-aat"
-  nonPreviewVaultName = "${var.core_product}-${var.env}"
-  vaultName = "${(var.env == "preview" || var.env == "spreview") ? local.previewVaultName : local.nonPreviewVaultName}"
-
-  s2sUrl = "http://rpe-service-auth-provider-${local.local_env}.service.${local.local_ase}.internal"
-  fees_register_url = "http://fees-register-api-${local.local_env}.service.${local.local_ase}.internal"
-
-  website_url = "http://${var.product}-api-${local.local_env}.service.${local.local_ase}.internal"
-
-  asp_name = "${var.env == "prod" ? "payment-api-prod" : "${var.core_product}-${var.env}"}"
-
-  sku_size = "${var.env == "prod" || var.env == "sprod" || var.env == "aat" ? "I2" : "I1"}"
+  s2sUrl = "http://rpe-service-auth-provider-${var.env}.service.core-compute-${var.env}.internal"
```

Comment:
@adusumillipraveen Why we have added this url , I believe its already present in values.yaml

Reply:
It is used in here https://github.com/hmcts/ccpay-payment-app/blob/2df7614f4b8b1781edaf24c551401bd709009a43/infrastructure/main.tf#L102 which still needs to be present.

Reply:
yes I can see its used in api policy, all good for me

Reply:
This is being used in API gateway I believe